development/input-app/components/input/Forwardrefinput/Forwardrefinput.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect,Ref, InputHTMLAttributes } from "react";
+import { ReuseInput } from "@locoworks/reusejs-react-input";
+import { useRef } from "react";
+import { HeadlessButton } from "@locoworks/reusejs-react-button";
+const Forwardrefinput = () => {
+    const sampleref = useRef<HTMLInputElement>(null);    
+
+       useEffect(() => {
```

Comment:
@Ayush2020012016 It's not clear if this file is formatted using prettier, indentation seems broken.